contrib/hadoop-store-builder/src/java/voldemort/store/readonly/fetcher/BasicFetchStrategy.java

```diff
@@ -64,21 +63,17 @@ public BasicFetchStrategy(HdfsFetcher fetcher, FileSystem fs, HdfsCopyStats stat
      * 'checkSumType' computed and returned. In case an error occurs during such
      * a copy, we do a retry for a maximum of NUM_RETRIES
      *
-     * @param fs
-     *            Filesystem used to copy the file
      * @param source
      *            Source path of the file to copy
      * @param dest
      *            Destination path of the file on the local machine
-     * @param stats
-     *            Stats for measuring the transfer progress
      * @param checkSumType
      *            Type of the Checksum to be computed for this file
      * @return A Checksum (generator) of type checkSumType which contains the
      *         computed checksum of the copied file
      * @throws IOException
      */
-    private CheckSum copyFileWithCheckSum(HdfsFile source, File dest, CheckSumType checkSumType) throws IOException {
+    private CheckSum copyFileWithCheckSum(HdfsFile source, File dest, CheckSumType checkSumType) throws Throwable {
```

Comment:
Sorry for the trouble caused here, from time to time my eclipse dies and I have to start the editor formatting from the first. It happens once every 6 months or so, this happened recently to my macbook and hence the code I wrote from macbook are mostly busted. Now I have fixed the formatting.